api/envoy/api/v2/route/route.proto

```diff
@@ -175,6 +175,9 @@ message Route {
 
     // Return an arbitrary HTTP response directly, without proxying.
     DirectResponseAction direct_response = 7;
+
+    // Return an noop action 
+    Noop  noop = 16;
```

Comment:
Would `Fallthru fallthru = 16;` be clearer? It's not that route is a nop that we execute and are done, we're actually going to keep looking for the real route entry (IIRC, correct me if wrong).